sys/Makefile.dep

```diff
@@ -16,7 +16,11 @@ ifneq (,$(filter i2c_scan,$(USEMODULE)))
 endif
 
 ifneq (,$(filter prng_fortuna,$(USEMODULE)))
-  CFLAGS += -DCRYPTO_AES
+  USEMODULE += crypto_aes
+endif
+
+ifneq (,$(filter crypto_%,$(USEMODULE)))
```

Comment:
Since there are two pseudomodules `crypto_aes_precalculated` and `crypto_aes_unroll` it might be useful to have a further dependency rule:
```
ifneq (,$(filter crypto_aes_%,$(USEMODULE)))
  USEMODULE += crypto_aes
endif
```

Reply:
Thanks for the suggestion, I added a commit for that.